server/views/formPages/incident/report-may-already-exist.html

```diff
@@ -0,0 +1,96 @@
+{% extends "../../partials/layout.html" %} 
+{% from "govuk/components/button/macro.njk" import govukButton %} 
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+
+{% set pageTitle = pageData.pageTitle %} 
+
+
+{% block content %}
+<div class="govuk-body">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      {% if errors.length > 0 %}
+      {{
+        govukErrorSummary({
+          titleText: 'There is a problem',
+          errorList: errors,
+          attributes: { 'data-qa-errors': true }
+        })
+      }}
+      {% endif %}
+      <h1 class="govuk-heading-xl mainHeading">{{ pageData.pageTitle }}</h1>
+      <p class="govuk-!-margin-bottom-1">
+        <span class="govuk-label--s">Prisoner:&nbsp;</span>
+        <span data-qa="offender-name">{{ pageData.offenderName }}</span>
+      </p>
+      <p>
+        A use of force incident for this prisoner has been reported as taking place on the same day. If the incident
+        you're reporting has already been reported, you do not need to create another report.
+      </p>
+
+      <table class="govuk-table" data-qa="table">
+        <caption class="govuk-table__caption govuk-table__caption--m">
+          Use of force incidents
+        </caption>
+
+        <thead class="govuk-table__head">
+          <tr class="govuk-table__row">
+            <th scope="col" class="govuk-table__header">Incident date and time</th>
+            <th scope="col" class="govuk-table__header">Location</th>
+            <th scope="col" class="govuk-table__header">Reporter</th>
+          </tr>
+        </thead>
+
+        <tbody class="govuk-table__body">
+          {% for report in pageData.reports %}
+          <tr class="govuk-table__row">
+            <td class="govuk-table__cell">{{report.date}}</td>
+            <td class="govuk-table__cell">{{report.location.userDescription}}</td>
```

Comment:
render userDescription rather than description as description is in upper case whereas as userDescription is in title case.